stored_requests/backends/http_fetcher/fetcher_test.go

```diff
@@ -249,7 +249,7 @@ func newAccountHandler(t *testing.T, expectAccIDs []string, jsonifier func(strin
 
 		for _, accID := range gotAccIDs {
 			if accID != "" {
-				accIDResponse[accID] = jsonifier(accID)
+				accIDResponse[accID] = json.RawMessage(`{"id":"` + accID + `"}`)
```

Comment:
Why not just use the existing `jsonifier` function that's been sent as parameter to `newAccountHandler`?

Reply:
We need a valid account json for this test case, the existing value was fails in json patching.  Removed jsonifier as I am already creating a valid json here and jsonifier adds escape characters to json string, which causes unmarshalling error.